src/Integration.Vsix/ErrorList/IssuesSnapshot.cs

```diff
@@ -63,6 +65,19 @@ internal interface IIssuesSnapshot : ITableEntriesSnapshot
         /// increment its version so the Error List recognises it as having changed
         /// </summary>
         void IncrementVersion();
+
+        /// <summary>
+        /// Create and return an updated version of an existing snapshot, where the
+        /// issues are the same but the source file has been renamed
+        /// </summary>
+        IIssuesSnapshot CreateUpdatedSnapshot(string newFilePath);
+
+        /// <summary>
+        /// Create and return an updated version of an existing snapshot, where the
+        /// set of issues are the same but their locations have changed (by the user editing the file)
+        /// </summary>
+        /// <remarks>The number and ordering of issues must be the same</remarks>
+        IIssuesSnapshot CreateUpdatedSnapshot(IEnumerable<IAnalysisIssueVisualization> updatedIssues);
```

Comment:
Do we use this somewhere? I think we used to, but there are no usages of this anymore.

Reply:
No. There are three static constructor methods for the snapshot that were all used by the TBIT. Only one of them should be used now.